src/plugins/data/common/field_formats/converters/source.test.ts

```diff
@@ -53,4 +53,156 @@ describe('Source Format', () => {
       '<span>{&quot;foo&quot;:&quot;bar&quot;,&quot;number&quot;:42,&quot;hello&quot;:&quot;&lt;h1&gt;World&lt;/h1&gt;&quot;,&quot;also&quot;:&quot;with \\&quot;quotes\\&quot; or &#39;single quotes&#39;&quot;}</span>'
     );
   });
+
+  describe('templateHtml rendering', () => {
```

Comment:
Adds in tests for testing the output of `htmlConvert` within `source.ts`, since this wasn't in there beforehand.